Documentation/contributor-guide/dependency_management.md

```diff
@@ -83,6 +84,11 @@ The plan is to remove the dependency on some grpc-go's experimental API firstly,
 
 `go.opentelemetry.io/otel` version update is indirectly blocked due to this gRPC issue. Please get more details in [pull/15810](https://github.com/etcd-io/etcd/pull/15810).
 
+## Rotation worksheet
+The dependabot scheduling interval is weekly; it means dependabot will automatically raise a bunch of PRs per week.
```

Comment:
Does it make sense to make it every other week ? There are tradeoffs either way, but it in my observation the number of PRs is uneven weekly.

Reply:
I do not see a obvious concern to set schedule to weekly.  Also it seems that there are [only three interval types](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval):
- daily
- weekly
- monthly